Box.V2/Managers/BoxMetadataManager.cs

```diff
@@ -113,12 +113,16 @@ public BoxMetadataManager(IBoxConfig config, IBoxService service, IBoxConverter
         /// <summary>
         /// Used to delete the template instance. To delete custom key:value pairs within a template instance, you should refer to the updating metadata section.
         /// </summary>
-        /// <param name="folderId">Id of folder</param>
-        /// <param name="scope">Scope name. Currently, only the enterprise scope is supported</param>
-        /// <param name="template">Metadata template name</param>
-        /// <returns>True if successful, false otherwise.</returns>
+        /// <param name="folderId">Id of folder.</param>
+        /// <param name="scope">Scope name. Currently, the only scopes support are enterprise and global.</param>
+        /// <param name="template">Metadata template name.</param>
+        /// <returns>Response to confirm the deletion of the template instance</returns>
         public async Task<bool> DeleteFolderMetadataAsync(string folderId, string scope, string template)
         {
+            folderId.ThrowIfNullOrWhiteSpace("folderId");
+            scope.ThrowIfNullOrWhiteSpace("scope");
```

Comment:
I am not sure if this kind of validation is necessary, since if the scope is null or space, the request will fail on the server-side anyway, what do you think @cburnette